### PR TITLE
Rails 6.0 support

### PR DESCRIPTION
**Pull request by jonwaghorn**
Adds support for use by Rails 6 projects

### CODE REVIEW

**Comment by jonwaghorn** [comment]
Fixes #55

**Comment by jonwaghorn** [comment]
Ugh, Rails 6 requires minimum Ruby 2.5.0...

File: lib/polo/sql_translator.rb [hunk 1]
```diff
@@ -133,14 +133,33 @@ def raw_sql_from_record(record)
       end
     end
 
+    # Internal: Returns an object's attribute definitions along with
+    # their set values (for Rails 6.0).
+    module ActiveRecordSix
+      def raw_sql_from_record(record)
+        values = record.send(:attributes_with_values, record.class.column_names)
```
Comment (mohammedyasir) [inline lib/polo/sql_translator.rb:140]: Actually the alternate for `attributes_with_values_for_create` should be `record.send(:attributes_with_values, record.send(:attributes_for_create, attribute_names))`

**Comment by jakeonfire** [comment]
this may help: https://github.com/IFTTT/polo/pull/59

**Comment by jakeonfire** [comment]
@jonwaghorn if you want to rebase master and get the build green we can approve this 👍

**Pull request merged**